src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -2202,19 +2202,69 @@ module Queries = struct
         Mina_lib.snark_pool mina |> Network_pool.Snark_pool.resource_pool
         |> Network_pool.Snark_pool.Resource_pool.all_completed_work )
 
+  (* Auxiliary function to fetch all pending work *)
+  let get_pending_work mina =
```

Comment:
I feel this is slightly misnamed now. The design has changed from the previous intention. Previously we obtained all pending work without fees, where as now it seems we obtain all work, including work that was previously completed in case another snark worker want to provide a better bid. I feel this change in intention should be documented and explained in this doc comment.

perhaps something like:
`get_pending_and_completed_work`